sponsorblockcast.sh

```diff
@@ -7,6 +7,18 @@ SBCSCANINTERVAL="${SBCSCANINTERVAL:-300}"
 SBCDIR="${SBCDIR:-/tmp/sponsorblockcast}"
 SBCCATEGORIES="${SBCCATEGORIES:-sponsor}"
 
+sysname=$(uname -s)
+if [ "$sysname" = "Darwin" ]; then
+  if which ggrep gsed > /dev/null; then
+    alias grep=ggrep
+    alias sed=gsed
+    shopt -s expand_aliases
+  else
+    echo >&2 $0 requires GNU grep and sed. Run \`brew install grep gnu-sed\`.
```

Comment:
`shellcheck` wants this $0 quoted, so might as well, though it shouldn't matter unless the path is weird.

Reply:
Got it. Done. 

Alternatively, I can change it to the literal `sponsorblockcast` if you prefer. It doesn't seem to appear elsewhere.